src/scripts/modules/popup.js

```diff
@@ -0,0 +1,64 @@
+const getPokemonInfo = async (id) => {
+  const getInfo = await fetch(`https://pokeapi.co/api/v2/pokemon/${id}/`);
+  const json = await getInfo.json();
+  return json;
+};
+
+export default (close) => {
+  const buttons = [...document.querySelectorAll('button')];
```

Comment:
[OPTIONAL] I think we could create a class for these buttons. That way we save space and avoid the if statement below

Reply:
Done!